custom_components/ojmicroline_thermostat/climate.py

```diff
@@ -89,13 +88,15 @@ class OJMicrolineThermostat(
 ):
     """OJMicrolineThermostat climate."""
 
-    _attr_hvac_modes = MODE_LIST
+    _attr_hvac_modes: ClassVar[list[HVACMode]] = [HVACMode.HEAT]
```

Comment:
Ruff insists on annotating this as a `ClassVar` because its default value is shared among all classes (and thus mutations to one affect all). This is never mutated anyway, so it's fine.